code/lib/cli/src/initiate.ts

```diff
@@ -260,7 +262,7 @@ export async function initiate(options: CommandOptions, pkg: Package): Promise<v
   logger.log(chalk.inverse(`\n ${welcomeMessage} \n`));
 
   if (!options.disableTelemetry) {
-    telemetry('init');
+    telemetry('init', {}, { stripMetadata: true });
```

Comment:
This was failing trying to find the metadata. I'm not really sure what changed, but we don't need it / can't find it anyway.